build.sbt

```diff
@@ -13,10 +13,11 @@ scalafmt: {
 
 val javaSpecificationVersion = "11"
 val scalacSilencerVersion    = "1.6.0"
+val scalaCompileVersion      = "2.13.1"
```

Comment:
typo: `scalaCompilerVersion` instead of `scalaCompileVersion`; I would recommend using `scalaVersion` instead if it doesn't create a conflict with an SBT key.

Reply:
it does